Sources/DifferentiableSwiftExamplesDocumentation/DifferentiableSwiftExamples.docc/Resources/Code/DifferentiableFunctions/DifferentiableFunctions-02-05.swift

```diff
@@ -0,0 +1,17 @@
+import _Differentiation
+
+func squared(_ input: Double) -> Double {
+    input * input
+}
+
+@derivative(of: squared)
+func vjpSquared(_ input: Double) -> (
+    value: Double,
+    pullback: (Double) -> Double
+) {
+    let output = squared(value)
+    func pullback(_ tangentVector: Double) -> Double {
+        return tangentVector * 2 * output)
```

Comment:
Parenthesis at the end should be removed.

Reply:
Also is this pullback correct? This looks like 2 x^2 dx

Reply:
You're right, this was incorrect. Both the parenthesis and the wrong derivative should be fixed now.